### DISCUSSION TITLE
git segment - disabling file status in specific repository

### DISCUSSION THREAD

**Discussion by Somft** [JanDeDobbeleer/oh-my-posh#595] [Troubleshoot] [answered]
Hi, 
I recently updated oh-my-posh to the v3 version. Up to now, i was able to disable file status in specific repository by using: 
`$GitPromptSettings.RepositoriesInWhichToDisableFileStatus += <path to repo>`
Unfortunately i don't know how to do it in git segment. 
When using poshgit it works. I would be really happy if i could use git segment which is easier to configure. 
Thanks for any help.

**Comment by JanDeDobbeleer** [comment, marked as answer]
@Somft I would in that case opt for the `poshgit` segment rather than the git one. That way you can keep your current config rather than working around it :-) See [here](https://ohmyposh.dev/docs/poshgit) for details.
